onnxruntime/core/providers/openvino/openvino_mo/openvino_mo.py

```diff
@@ -0,0 +1,447 @@
+#!/usr/bin/env python3
+
+"""
+  Copyright(C) 2019 Intel Corporation
+  Licensed under the MIT License
+"""
+
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+import os
+ov_root = os.environ['INTEL_CVSDK_DIR']
```

Comment:
how does this work if you built openvino from source?

Reply:
As we disabled building from source for now, this is hardcoded. This will change once we enable building from source